code/src/components/TotalTodos.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+const Image = styled.img`
+  height: 200px;
+`
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  background-color: tomato;
+  color: white;
+  font-size: 24px;
+`
+const Title = styled.div`
+
+`
+const Total = styled.p`
+
+`
```

Comment:
I don't think you need these. If you don't style them, you could just have a div and a p-tag in your return statement